test/spec/validations/crossing_ways.js

```diff
@@ -373,6 +373,11 @@ describe('iD.validations.crossing_ways', function () {
         verifySingleCrossingIssue(validate(), null);
     });
 
+    it('flags road bridge crossing road on the same layer', function() {
```

Comment:
I'm not sure about the test positions in this file, but here seems to be the most sensible place.